### ISSUE TITLE
When debugger is attached, Unity project will not start playing

### ISSUE THREAD

**Issue by brokenbeta** [Unity-Technologies/vscode-unity-debug#133]
## Expected Behavior
You should be able to start a project while the debugger is attached.

## Current Behavior
The plugin attaches ok to Unity, but then when I press play in Unity, the project will never start. Unity locks up and I get a spinning pinwheel. If I stop the debugger, the Unity project suddenly starts.

## Steps to Reproduce (for bugs)
1. Attach debugger (F5)
2. Press play in Unity
Pinwheel appears and editor locks up

## Your Environment
* Unity Version: 2018.3.10f1
* VS Code Version: 1.32.3
* Version of the debugger tool: 2.7.2
* Operating System and version: Mac OS X 10.12.6

**Comment by SirIntruder** [comment]
yup, that's annoying... I've found "restart" button in vscode debug panel to work as well, less annoying than stopping/starting session from scratch

**Comment by sommerper** [comment]
Sorry to say that this makes the plugin pretty much useless for me. Restart is not an option since it misses out on a lot of initialisation code. :(

**Comment by lVlrChris** [comment]
I'm having a similar problem, however my editor does not seem to freeze. When I start the debugger in Visual Studio Code, it attaches to the Unity editor process but Unity will not enter play mode. Restarting did not seem to resolve the problem.
When I start the debugger I get the following logs from the debug console:
```
UnityDebug: Initializing
UnityDebug: Searching for Unity process 'Unity Editor (Unity) (5464)'
UnityDebug: Attached to Unity process 'Unity Editor (Unity)' (5464)
```
According to the logs the debugger should be attached correctly. Even if I would close Unity after starting the debugger it would show: `UnityDebug: Disconnected`, so it seems to be connected to something.
Perhaps something to note is that I've had this problem on three devices so far.

I'll provide my information as well:

- Unity Version: 2018.3.14f1
- VS Code Version: 1.39.2
- Version of the debugger tool: 2.7.2
- Operating System and version: Windows 10 Pro

**Comment by mgarbade** [comment]
deleting all breakpoints helped for me.
